### PR TITLE
Added exchange and network information

### PR DESCRIPTION
**Pull request by zihadbinjahangir**
Exchange information for getting information about symbols category. The network information file is fo getting the network connection

### CODE REVIEW

File: network/network_status.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        while True:
+            try:
+                print("Trying to connect...")
+                client = Client(self.api_key, self.api_secret)
+                server_time = client.get_server_time()
+                if server_time:
+                    print("Connection to Binance API is working.")
+                    return client
+            except Exception as e:
+                print(f"Connection to Binance API failed with error: {e}")
```
Comment (sushen) [inline network/network_status.py:24]: Separate Every Exception with its name.

**Comment by zihadbinjahangir** [comment]
Changed One exception to multiple exception.

File: network/network_status.py [hunk 2]
```diff
@@ -18,10 +20,22 @@ def connect(self):
...
+            except binance.exceptions.BinanceAPIException as e:
+                print(f"Binance API exception: {e}")
+                time.sleep(60)
+                continue
+            except binance.exceptions.BinanceOrderException as e:
+                print(f"Binance order exception: {e}")
+                time.sleep(60)
+                continue
+            except Exception as e:
+                print(f"Unknown exception: {e}")
```
Comment (sushen) [inline network/network_status.py]: The hole point is to get exception.

**Comment by zihadbinjahangir** [comment]
Changed One exception to multiple exception.

**Comment by zihadbinjahangir** [comment]
Changed One exception to multiple exceptions.

**Review by sushen — APPROVED** [review]

**Pull request merged**